xbmc/interfaces/json-rpc/AudioLibrary.cpp

```diff
@@ -1396,20 +1393,17 @@ JSONRPC_STATUS CAudioLibrary::RefreshAlbum(const std::string& method,
   if (!musicdatabase.Open())
     return InternalError;
 
-  CMusicDbUrl musicUrl;
-  if (!musicUrl.FromString("musicdb://albums/"))
-    return InvalidParams;
-
-  //check if albumID is a valid one
+  // Check if albumID is a valid one
   CAlbum album;
   if (!musicdatabase.GetAlbum(albumID, album, false))
     return InvalidParams;
 
-  //set the album id on the musicdb url
+  // Set the album id on the musicdb url
+  CMusicDbUrl musicUrl;
   musicUrl.AddOption("albumid", albumID);
 
-  //Start rescraping additional information for the given album
-  std::string cmd = StringUtils::Format("musiclibrary.refreshalbum({})",
+  // Start rescraping additional information for the given album
+  const std::string cmd = StringUtils::Format("musiclibrary.refreshalbum({})",
                                         StringUtils::Paramify(musicUrl.ToString()));
```

Comment:
wrong indentation - jenkins will tell you.

Reply:
just one question, if i apply the clang format it'll format the whole cpp file, is that ok ?

Reply:
depends on how you invoke clang-format.
'git clang-format HEAD~1' for example only fixes the issues with latest commit.
'clang-format -i filename.cpp' will reformat the whole given file.

Reply:
But you can also wait for Jenkins producing the proper diff for you.

Reply:
ah ok thanks for the hints, didnt know about applying formats directly on git commits, always applied to the whole file directly on vscode or through jenkins patch :)